src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
```

Comment:
The check for property presence using 'Object.prototype.hasOwnProperty.call' is correct and ensures that the property comes directly from the object rather than its prototype chain.